app/services/taxonomy/taxon_parents_initializer.rb

```diff
@@ -0,0 +1,57 @@
+module Taxonomy
+  # Copies the parent links to the new field taxon_parents
+  class TaxonParentsInitializer
```

Comment:
Need a different name I reckon - 'initializer' is a little overloaded in Rails land. 

Could call this something like `PopulateTaxonParents` (or `PopulateParentTaxons` if we go with my naming suggestion above 😃 ).